graphql/database/missions/MissionModel.js

```diff
@@ -0,0 +1,98 @@
+import BaseModel from '../base/BaseModel'
+import types from '../fieldTypes'
+import tableNames from '../tables'
+import { MISSION } from '../constants'
+import { permissionAuthorizers } from '../../utils/authorization-helpers'
+
+/*
+ * @extends BaseModel
+ */
+class Mission extends BaseModel {
+  static get name() {
+    return MISSION
+  }
+
+  static get hashKey() {
+    return 'id'
+  }
+
+  static get tableName() {
+    return tableNames.missions
+  }
+
+  static get schema() {
+    return {
+      id: types
+        .string()
+        .required()
+        .description(`The unique mission id`), // not sure how to nanoid this value
+      squadName: types
+        .string()
+        .required()
+        .description(`user entered squad type`),
+      created: types
+        .string()
+        .isoDate()
+        .required(),
+      started: types
+        .string()
+        .isoDate()
+        .description('timestamp from when mission is active'),
+      completed: types
+        .string()
+        .isoDate()
+        .description('timestamp from when mission is completed'),
+      tabGoal: types
+        .number()
+        .integer()
+        .description(`number of tabs for goal`),
+      missionType: types
```

Comment:
Remove for now?

Reply:
keeping with a default value so we don't have to update these docs later.  I'll just not send down from graphql for now if that works